tests/processes/ProcessingTest.py

```diff
@@ -234,6 +241,7 @@ def test_dead_processes(self):
             ctrlq, {}, {}, {},
             lambda *args: self.queue.put(args[2]),
             Section(""),
+            None,
```

Comment:
With the result not being fixed during/after the first run to ensure results are not dropped? Yeah, that's a very good idea :+1: